falcon/testing/client.py

```diff
@@ -275,8 +276,14 @@ def simulate_request(app, method='GET', path='/', query_string=None,
                 environ (default: ``None``). This can be used to test
                 high-performance file transmission when `resp.stream` is
                 set to a file-like object.
+            host(str): A string to use for the hostname part of the fully
```

Comment:
I these additions look OK, I will update copy-paste them to specialized ``simulate_*`` methods.

Reply:
LGTM 👍

Reply:
Done with copy-paste now.